.env.template

```diff
@@ -1,3 +0,0 @@
-BOT_TOKEN=yourbottoken
```

Comment:
Why was this all removed?

Reply:
That was another mistake, my bad. I need to get better with this tech stack. I'm stepping down responsibility until I have a few more solid commits under my belt.

Reply:
It's not a big deal. I think you've got the environment variables and line endings worked out. Just make sure to have a very good idea of what's going into every commit by reading over the diffs.